packages/eth2.0-utils/src/yaml/int.ts

```diff
@@ -119,43 +118,48 @@ function constructYamlInteger(data: string): BN {
     ch = value[0];
   }
 
-  if (value === "0") return new BN(0);
+  if (value === "0") return 0n;
 
   if (ch === "0") {
-    if (value[1] === "b") return (new BN(value.slice(2), 2)).muln(sign);
-    if (value[1] === "x") return (new BN(value, 16)).muln(sign);
-    return (new BN(value, 8)).muln(sign);
+    if (value[1] === "b") return BigInt("0b" + value.slice(2)) * BigInt(sign);
+    if (value[1] === "x") return BigInt( value) * BigInt(sign);
```

Comment:
spacing,
I think these two if statements can be collapsed to
`if (value[1] === "b" || value[1] === "x") return BigInt(value) * BigInt(sign);`